apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -1,10 +1,10 @@
 import { createTheme, Theme } from '@mui/material'
+import { mergeTheme } from 'synapse-react-client/dist/utils/theme'
 
 export const latoFont = ['Lato', 'Roboto', 'Helvetica', 'Arial'].join(',')
 
-const themeObject = {
+export const themeOptions = mergeTheme({
```

Comment:
The themeOptions will be these theme overrides merged with the default theme options defined in Synapse-React-Client.